### PR TITLE
Fix and polish zh-Hans translation

### PR DESCRIPTION
**Pull request by errlot**

### CODE REVIEW

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 1]
```diff
@@ -1366,14 +1366,14 @@
 			"loadingData": "正在加载数据...",
 			"rateLimited": "超出 GitHub 查询限制。请稍候。",
 			"similarIssues": "类似的问题",
-			"open": "打开",
+			"open": "未结",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Denoting the status of an issue, should be an adjective.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 2]
```diff
@@ -1778,7 +1778,7 @@
 			"patchedWindowTitle": "[不受支持]",
 			"userIsAdmin": "[管理员]",
 			"userIsSudo": "[超级用户]",
-			"devExtensionWindowTitlePrefix": "[扩展开发主机]"
+			"devExtensionWindowTitlePrefix": "[扩展开发宿主]"
 		},
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: "机" in "主机" means a machine, I don't think it's suitable to call a Extension Development Host window a 主机. Same applies to "扩展主机" (Extension host), it's a process though.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 3]
```diff
@@ -2057,7 +2057,7 @@
 			"splitRight": "向右拆分",
 			"close": "关闭",
 			"closeOthers": "关闭其他",
-			"closeRight": "关闭到右侧",
+			"closeRight": "关闭右侧",
 			"closeAllSaved": "关闭已保存",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Close [tabs] to the right. "tabs" is omitted

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 4]
```diff
@@ -3542,7 +3542,7 @@
 			"createNewFolder": "新建文件夹",
 			"newUntitledFile": "新的无标题文件",
 			"deleteButtonLabelRecycleBin": "移动到回收站(&&M)",
-			"deleteButtonLabelTrash": "移动到回收站(&&M)",
+			"deleteButtonLabelTrash": "移动到废纸篓(&&M)",
 			"deleteButtonLabel": "删除(&&D)",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: "废纸篓" is the official translation of Trash in macOS

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 5]
```diff
@@ -3622,7 +3622,7 @@
 			"dirtyCounter": "{0} 个未保存"
 		},
 		"vs/workbench/contrib/files/common/editors/fileEditorInput": {
-			"orphanedFile": "{0} (从磁盘中删除)",
+			"orphanedFile": "{0} (磁盘上已删除)",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: The file's status is "deleted from disk"

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 6]
```diff
@@ -3795,7 +3795,7 @@
 			"label": "问题",
 			"tooltip.1": "此文件存在 1 个问题",
 			"tooltip.N": "此文件存在 {0} 个问题",
-			"markers.showOnFile": "显示文件和文件夹上的错误和警告。"
+			"markers.showOnFile": "在文件和文件夹上显示错误和警告。"
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: This settings control whether it shows errors/warnings on File/Folder items in File Explorer. Its shows them regardlessly in Problem panel. Same applies to the Outline one below.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 7]
```diff
@@ -38,7 +38,7 @@
 			"quickInput.visibleCount": "{0} 个结果"
 		},
 		"vs/base/browser/ui/aria/aria": {
-			"repeated": "{0} (已再次发生)",
+			"repeated": "{0} (再次发生)",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Announced by screen reader when the message is the same as the previous one.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 8]
```diff
@@ -534,17 +534,17 @@
...
+			"smartSelect.shrink": "缩小选择范围",
 			"miSmartSelectShrink": "缩小选定范围(&&S)"
 		},
 		"vs/editor/contrib/suggest/suggestController": {
 			"arai.alert.snippet": "Accepting '{0}' made {1} additional edits",
 			"suggest.trigger.label": "触发建议"
 		},
 		"vs/editor/contrib/tokenization/tokenization": {
-			"forceRetokenize": "开发人员: 强制重新令牌化"
+			"forceRetokenize": "开发者: 强制重新进行标记"
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: I see "Developer" is translated to "开发者" in most occurrences.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 9]
```diff
@@ -509,12 +509,12 @@
 			"miInsertCursorAtEndOfEachLineSelected": "在行尾添加光标(&&U)",
 			"mutlicursor.addCursorsToBottom": "在底部添加光标",
 			"mutlicursor.addCursorsToTop": "在顶部添加光标",
-			"addSelectionToNextFindMatch": "将选择添加到下一个查找匹配项",
+			"addSelectionToNextFindMatch": "将下一个查找匹配项添加到选择",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: My translation looks weird based on the source, but the product does exactly like this, trust me.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 10]
```diff
@@ -849,14 +849,14 @@
...
 			"invalidManifest": "扩展无效: package.json 不是 JSON 文件。",
-			"incompatible": "无法安装扩展名'{0}'，因为它不兼容 VS Code '{1}'。",
-			"restartCode": "请在重新安装{0}之前重新启动 VS Code。",
+			"incompatible": "无法安装扩展“{0}”，因为它不兼容 VS Code “{1}”。",
+			"restartCode": "请在重新安装 {0} 之前重新启动 VS Code。",
 			"malicious extension": "无法安装此扩展，它被报告存在问题。",
 			"notFoundCompatibleDependency": "无法安装。找不到与 VS Code 当前版本 ({1}) 兼容的扩展“{0}”。",
 			"notSupportedUIExtension": "无法安装扩展 {0}，因为不支持 UI 扩展",
-			"MarketPlaceDisabled": "商店未启用",
+			"MarketPlaceDisabled": "市场未启用",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: All Marketplace->"市场"

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 11]
```diff
@@ -2139,7 +2139,7 @@
 			"focusPanel": "聚焦到面板中",
 			"toggledPanelPosition": "切换面板位置",
 			"moveToRight": "将面板移动到右侧",
-			"moveToBottom": "将面板移至底部",
+			"moveToBottom": "将面板移动到底部",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Consistency

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 12]
```diff
@@ -2692,7 +2692,7 @@
 			"message": "消息",
 			"condition": "条件",
 			"breakpointHasConditionEnabled": "此{0}的{1}将在删除后丢失。请考虑仅禁用此{0}。",
-			"removeLogPoint": "删除 {0}",
+			"removeLogPoint": "删除{0}",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: {0} is a Chinese word all the time, looks very bad with the extra space.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 13]
```diff
@@ -3102,13 +3102,13 @@
 		"vs/workbench/contrib/extensions/electron-browser/extensionsActions": {
 			"download": "手动下载",
 			"install vsix": "下载后，请手动安装“{0}”的 VSIX。",
-			"noOfYearsAgo": "{0} 年前",
+			"noOfYearsAgo": "{0}年前",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Follows the translation of Windows, no space when it comes to date and time.

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 14]
```diff
@@ -3515,19 +3515,19 @@
 			"retry": "重试",
 			"discard": "放弃",
 			"readonlySaveErrorAdmin": "无法保存“{0}”: 文件写保护。选择“以管理员身份覆盖”可作为管理员重试。 ",
-			"readonlySaveErrorSudo": "无法保存“{0}”：文件被写保护。选择“Overwrite as Sudo”以超级用户身份重试。",
+			"readonlySaveErrorSudo": "无法保存“{0}”: 文件写保护。选择“使用 sudo 覆盖”可作为超级用户重试。",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: `sudo` is a program

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 15]
```diff
@@ -2496,9 +2496,9 @@
 			"configuredOn": "已配置编辑器对屏幕阅读器进行永久优化 — 您可以更改 \"editor.accessibilitySupport\" 设置进行调整。",
 			"configuredOff": "编辑器被配置为不对屏幕阅读器的使用进行优化。",
 			"tabFocusModeOnMsg": "在当前编辑器中按 Tab 会将焦点移动到下一个可聚焦的元素。通过按 {0} 切换此行为。",
-			"tabFocusModeOnMsgNoKb": "在当前编辑器中按 Tab 会将焦点移动到下一个可聚焦的元素。当前无法通过键绑定触发命令 {0}。",
+			"tabFocusModeOnMsgNoKb": "在当前编辑器中按 Tab 会将焦点移动到下一个可聚焦的元素。当前无法通过按键绑定触发命令 {0}。",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: all "Keybinding"->"按键绑定"

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 16]
```diff
@@ -4866,8 +4866,8 @@
 			"workbench.action.terminal.resizePaneUp": "向上调整窗格大小",
 			"workbench.action.terminal.resizePaneDown": "向下调整窗格大小",
 			"workbench.action.terminal.focus": "聚焦到终端",
-			"workbench.action.terminal.focusNext": "聚焦于下一终端",
-			"workbench.action.terminal.focusPrevious": "聚焦于上一终端",
+			"workbench.action.terminal.focusNext": "聚焦到下一终端",
+			"workbench.action.terminal.focusPrevious": "聚焦到上一终端",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Consistency

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 17]
```diff
@@ -4970,7 +4970,7 @@
 			"installColorThemes": "安装其他颜色主题...",
 			"themes.selectTheme": "选择颜色主题 (按上下箭头键预览)",
 			"selectIconTheme.label": "文件图标主题",
-			"noIconThemeLabel": "没有",
+			"noIconThemeLabel": "无",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Context
image

File: i18n/vscode-language-pack-zh-hans/translations/main.i18n.json [hunk 18]
```diff
@@ -5115,7 +5115,7 @@
 			"welcomePage.keymapNotFound": "找不到 ID 为 {1} 的 {0} 键盘快捷方式。",
 			"welcome.title": "欢迎使用",
 			"welcomePage.openFolderWithPath": "打开路径为 {1} 的文件夹 {0}",
-			"welcomePage.extensionListSeparator": ", ",
+			"welcomePage.extensionListSeparator": "、",
```
Comment (errlot) [inline i18n/vscode-language-pack-zh-hans/translations/main.i18n.json]: Context
image

**Comment by lukaschal** [comment]
On review.

**Comment by lukaschal** [comment]
Fixed as suggested in a MLCP.

**Comment by lukaschal** [comment]
@errlot Please verify and close this issue.

**Comment by errlot** [comment]
Nope, not a single one is fixed in the latest commit after my quick glance.
Also, it's annoying to "verify" when https://github.com/Microsoft/vscode-loc/issues/84 is still open.

**Pull request closed without merging**